File: 2015.01.10_ABC_Powtorka_ObjectiveC/Powtorka/ViewController.m
//
//  ViewController.m
//  Powtorka
//
//  Created by IIUJ on 10.01.2015.
//  Copyright (c) 2015 IIUJ. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.display.text = @"nic";
    [self.buttonP2 addTarget:self action:@selector(updateDisplay:) forControlEvents:UIControlEventTouchUpInside];
    // Do any additional setup after loading the view, typically from a nib.
}
- (IBAction)updateDisplay:(UIButton *)sender {
    
    self.display.text = sender.titleLabel.text;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 2015.03.01_Swift1_wersja2/test2/ViewController.swift
//
//  ViewController.swift
//  test2
//
//  Created by IIUJ on 01.03.2015.
//  Copyright (c) 2015 IIUJ. All rights reserved.
//

import UIKit

class Person {
    var residence: Residence?
}

class Residence {
    var numberOfRooms = 1
}


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    let john = Person()
    let roomCount = john.residence?.numberOfRooms
    let x = 5
    println(x)
    
    
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

